### DISCUSSION TITLE
Viewing text from pipe redirected output opens the `LazyVim` dashboard

### DISCUSSION THREAD

**Discussion by surmish** [LazyVim/LazyVim#3125] [Q&A] [answered]
When I run something like this
```
echo "hello" | nvim - 
```
is there a way to not open `LazyVim` dashboard? I have to do `:bn` or `Ctrl-L` to move to the next buffer with `hello` in it.

**Comment by dpetka2001** [comment, marked as answer]
When #3107 gets merged, this won't happen.
